Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs

```diff
@@ -64,7 +67,8 @@ data UpdateProfileReq = UpdateProfileReq
     language :: Maybe Maps.Language,
     gender :: Maybe Person.Gender,
     bundleVersion :: Maybe Version,
-    clientVersion :: Maybe Version
+    clientVersion :: Maybe Version,
+    disability :: Maybe PersonDisability.DisabilityItem
```

Comment:
`DisabilityItem` type is just request type. It should be defined here only instead of Domain object.